src/styles/blocks/catalog.scss

```diff
@@ -18,6 +18,13 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: 300ms;
```

Comment:
please create the css variable for transition time because you're using it few times